command/sub8_launch/launch/subsystems/cameras.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
   <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/front_cameras.launch" />
-  <!-- include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" / -->
+  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" />
```

Comment:
Its based in the ID camera. I guess it uses that to find it on the network somehow